v23.1/crdb-internal.md

```diff
@@ -268,6 +268,28 @@ SELECT * FROM crdb_internal.cluster_contention_events;
 (9 rows)
 ~~~
 
+#### View the tables/indexes with the most time under contention
+
+To view the [tables](create-table.html) and [indexes](indexes.html) with the most cumulative time under [contention](performance-best-practices-overview.html#transaction-contention) since the last server restart, run the following query:
```

Comment:
The contention events are based on tracing. Tracing by default is only enabled for a very small percentage of requests by default. It probably should mention this to avoid confusion. Related issue: https://github.com/cockroachdb/cockroach/issues/96453

Reply:
thanks @j82w !

I've added a note to that effect (which also appears at https://www.cockroachlabs.com/docs/dev/ui-insights-page#transaction-executions-view) that says 

> The default tracing behavior captures a small percent of transactions so not all contention events will be recorded. When investigating transaction contention, you can set the sql.trace.txn.enable_threshold [cluster setting](https://www.cockroachlabs.com/docs/v23.1/cluster-settings) to always capture contention events.

I've also filed https://cockroachlabs.atlassian.net/browse/DOC-6967 to add that info to all of the `crdb_internal` things that look at contention, since presumably they all use that setting